cmd/service_bootstrap_desktop.go

```diff
@@ -80,8 +76,18 @@ func (di *Dependencies) bootstrapServiceOpenvpn(nodeOptions node.Options) {
 		currentLocation := market.Location{Country: location.Country}
 		transportOptions := serviceOptions.(openvpn_service.Options)
 
+		mapPort := func() func() {
```

Comment:
This makes me uncomfortable. Since we're already using this in two places(and probably more to come)  I think it makes sense to go a step further, and maybe have a func like this in the mapping package:

```golang
func GetPortMappingFunc(pubIP,outIP, protocol,port, description) func() {
    if pubIP != outIP {
		return mapping.PortMapping(protocol, port, description)
    }
    return func() {}
}
```

In that case, it will be more concise to inject these to the corresponding services, as the map port would become something like this:

```golang
mapPort := func() {
    return mapping.GetPortMappingFunc(location.OutIP, locationPubIp, ....)
} 
```

Reply:
will do.